CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## 2.2.0
```

Comment:
This is a bug fix release, the release number should be `2.1.1`

Reply:
The bidirectional PV feels more like a minor functional change. @bianchi2 what do you think?

Reply:
Still this is a bug fix. This fixes the problem of not removing pv in uninstall in some cases.

Reply:
It's just a smarter way to create volumes - we make sure that the volume we create for a DC app does not get bound to elastic search (or any other 3rd party claim)